include/Nodes1DProvisioner.hpp

```diff
@@ -96,52 +146,29 @@ namespace blitzdg {
       Nodes1DProvisioner(index_type NOrder, index_type NumElements, real_type xmin, real_type xmax);
 
       /**
-       * Builds the nodes and all geometric factors for each element.
-       */
-      void buildNodes();
-
-      /**
-       * Builds the global connectivity matrices (EToE, EToF) for 1D grid
-       * based on the EToV (Element-to-Vertex) matrix.
-       */
-      void buildConnectivityMatrices();
-
-      /**
-       *  Builds the face mask that when applied to volume nodes gives the surface nodes.
+       * Deleted copy constructor.
        */
-      void buildFaceMask();
+      Nodes1DProvisioner(const Nodes1DProvisioner&) = delete;
 
       /**
-       * Builds the differentiation matrix Dr on the standard element.
+       * Deleted copy assignment operator.
        */
-      void buildDr();
+      Nodes1DProvisioner& operator=(const Nodes1DProvisioner&) = delete;
 
       /**
-       * Computes the Vandermonde matrix which maps modal coefficients to nodal values.
+       * Default move constructor.
```

Comment:
If this is default, do we need to explicitly write it out here?

Reply:
I can change the comment, but the defaulted move constructor definition needs to be part of the class definition.